src/renderer/components/Editor/SquareBracket.tsx

```diff
@@ -6,9 +6,10 @@ import SquareBracketHover from './SquareBracketHover';
 interface Props {
   isLast: boolean;
   isTakeGroupOpened: boolean;
+  takeHeight: number;
```

Comment:
Make it clear that this is in `px`. Kind of hacky that we have to do this in JS in the first place but we have other JS things that should be CSS so I'll give it a pass